generator/tests/testdata/golden/php/default/endpoints_google_owned.golden

```diff
@@ -172,6 +172,7 @@ class HelloGreetingCollection extends \Google\Collection
   protected $collection_key = 'items';
   protected $itemsType = HelloGreeting::class;
   protected $itemsDataType = 'array';
+  public $items;
```

Comment:
Is the usage of public keyword intentional in all files?
I am not sure about how these templates are used, but it looked odd that only new props are made public.

Reply:
Because when PHP declares properties on the fly, they are public. So we have to declare them as public to maintain backwards compatibility